purchases/src/main/kotlin/com/revenuecat/purchases/galaxy/GalaxyBillingWrapper.kt

```diff
@@ -112,7 +128,101 @@ internal class GalaxyBillingWrapper(
         shouldConsume: Boolean,
         initiationSource: PostReceiptInitiationSource,
     ) {
-        warnLog { "Unimplemented: GalaxyBillingWrapper.consumeAndSave" }
+        if (!finishTransactions || purchase.type == ProductType.UNKNOWN) return
```

Comment:
In these cases, I think we should still call `addSuccessfullyPostedToken` so we're not posting the same tokens over and over? As long as it's not in a pending state though.

Reply:
Updated in 7dbaab997a9c8b8cde502f370d69d72cbaf0b065 to store the token when the product type is Unknown or when observer mode is enabled 👍